pkg/generator.go

```diff
@@ -508,6 +508,8 @@ func (g *Generator) renderType(ctx context.Context, typ types.Type) string {
 			args = append(args, g.renderType(ctx, arg))
 		}
 		return fmt.Sprintf("%s[%s]", name, strings.Join(args, ","))
+	case *types.Alias:
```

Comment:
I'm actually surprised that CI/CD is not failing in the go1.21 test. This didn't exist until 1.22 so technically this won't compile if someone tries to build with 1.21.

If I'm understanding that correctly, we will need to make sure people understand. But I'd also like to figure out why CICD didn't fail.

Reply:
I see, go will download at least 1.22 because of our go.mod

```
$ /opt/homebrew/opt/go@1.21/bin/go version
go: downloading go1.22.0 (darwin/arm64)
go version go1.22.0 darwin/arm64
```

Also 1.21 is not supported anymore anyways, so we can probably remove it from the matrix and figure out how to get 1.23 to work in the test matrix.

```
Warning: go@1.21 has been deprecated because it is not supported upstream! It will be disabled on 2025-08-16.
```

Reply:
I did a CI test with go 1.23 enabled, but it requires the lastest golangci-lint, but installing it forces the library's go.mod to go 1.23.